docs/static-pvc.md

```diff
@@ -0,0 +1,115 @@
+# Static PVC with ceph-csi
+
+- [Static PVC with ceph-csi](#static-pvc-with-ceph-csi)
+  - [RBD static PVC](#rbd-static-pvc)
+    - [Create RBD image](#create-rbd-image)
+    - [Create RBD static PV](#create-rbd-static-pv)
+    - [RBD Volume Attributes in PV](#rbd-volume-attributes-in-pv)
+    - [Create RBD static PVC](#create-rbd-static-pvc)
+
+This document outlines how to create static PV and static PVC from existing rbd image
+
+**warning** static PVC can be created, deleted, mounted and unmounted but
+doesn't support other operations like (snapshot, resize, etc)
+
+## RBD static PVC
+
+RBD images created manually can be mounted and unmounted to an app, below step
+shows how to create a rbd image, static PV, static PVC
+
+### Create RBD image
```

Comment:
Do we really want to encourage manual image creation to plug into a static PV or should this really be focused on recovery. i.e. how to find an existing RBD image for a given PV?

Reply:
yes, we can add it also, the main idea for this doc is to use the rbd image (which may not be created by CSI driver itself) as an enhancement we can do this also.

currently reworded the statement, we can add recovery steps later.